internal/sirius/get_caseload_list.go

```diff
@@ -0,0 +1,70 @@
+package sirius
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type AnnualReport struct {
+	DueDate string `json:"dueDate"`
+}
+
+type Order struct {
+	OrderId            int          `json:"id"`
```

Comment:
It is not necessary to append the name of the struct to its parameters. We know the `id` parameter on an `Order` is the id of the order, so we don't need to call it `OrderId`. Same goes for all the client ones.

Reply:
Changed them all